TestProjects/Playground/Playground.Core/ViewModels/MixedNavMasterDetailViewModel.cs

```diff
@@ -63,14 +57,15 @@ public MixedNavMasterDetailViewModel(IMvxNavigationService navigationService)
                         return;
 
                     var vmType = item.ViewModelType;
-                    if(Xamarin.Forms.Application.Current.MainPage is MasterDetailPage masterDetailPage)
-                    {
-                        masterDetailPage.IsPresented = false;
-                    }
-                    else if(Xamarin.Forms.Application.Current.MainPage is NavigationPage navigationPage && navigationPage.CurrentPage is MasterDetailPage nestedMasterDetail)
-                    {
-                        nestedMasterDetail.IsPresented = false;
-                    }
+                    // TODO: This logic doesn't belong here
```

Comment:
Does the sample works without this code? Do you know where to put it if it doesn't belong there anymore?

Reply:
Ah good question. The sample definitely runs - setting the IsPresented to false controls whether the master page is visible. I've left the TODO in there as the samples could probably do with further refining, particularly once we've fixed the Forms implementation for MasterDetails (which doesn't really work at the moment)

Reply:
I think we should fix this together with the implementation in a new PR.